test/test_functional_tensor_cleanup.py

```diff
@@ -0,0 +1,145 @@
+            scripted_fn(tensor, 2.)
+        self.assertTrue('Tensor is not a torch image.' in str(context.exception))
+
+    def test_center_crop(self):
+        scripted_fn = torch.jit.script(F_t.center_crop)
+        shape = (100,)
+        tensor = torch.rand(*shape, dtype=torch.float, device=self.device)
+        with self.assertRaises(Exception) as context:
+            scripted_fn(tensor, [10, 11])
+        self.assertTrue('Tensor is not a torch image.' in str(context.exception))
+
+    def test_five_crop(self):
+        scripted_fn = torch.jit.script(F_t.five_crop)
+        shape = (100,)
+        tensor = torch.rand(*shape, dtype=torch.float, device=self.device)
+        with self.assertRaises(Exception) as context:
+            scripted_fn(tensor, [10, 11])
+        self.assertTrue('Tensor is not a torch image.' in str(context.exception))
+
+    def test_ten_crop(self):
+        scripted_fn = torch.jit.script(F_t.ten_crop)
+        shape = (100,)
+        tensor = torch.rand(*shape, dtype=torch.float, device=self.device)
+        with self.assertRaises(Exception) as context:
+            scripted_fn(tensor, [10, 11])
+        self.assertTrue('Tensor is not a torch image.' in str(context.exception))
+
+    def test_pad(self):
+        scripted_fn = torch.jit.script(F_t.pad)
+        shape = (100,)
+        tensor = torch.rand(*shape, dtype=torch.float, device=self.device)
+        # config = {"padding_mode": "constant", "fill": 0}
+        with self.assertRaises(Exception) as context:
+            scripted_fn(tensor, [2, ], 2, "constant")
+        self.assertTrue('Tensor is not a torch image.' in str(context.exception))
+
+    def test_resize(self):
+        scripted_fn = torch.jit.script(F_t.resize)
+        shape = (100,)
+        tensor = torch.rand(*shape, dtype=torch.float, device=self.device)
+        with self.assertRaises(Exception) as context:
+            scripted_fn(tensor, [10, 11])
+        self.assertTrue('Tensor is not a torch image.' in str(context.exception))
+
+    def test_perspective(self):
+        scripted_fn = torch.jit.script(F_t.perspective)
+        shape = (100,)
+        tensor = torch.rand(*shape, dtype=torch.float, device=self.device)
+        with self.assertRaises(Exception) as context:
+            scripted_fn(tensor, [0.2, ])
+        self.assertTrue('Tensor is not a torch image.' in str(context.exception))
+
+    def test_gaussian_blur(self):
+        scripted_fn = torch.jit.script(F_t.gaussian_blur)
+        shape = (100,)
+        tensor = torch.rand(*shape, dtype=torch.float, device=self.device)
+        with self.assertRaises(Exception) as context:
+            scripted_fn(tensor, (2, 2), (0.7, 0.5))
+        self.assertTrue('Tensor is not a torch image.' in str(context.exception))
+
+
+@unittest.skipIf(not torch.cuda.is_available(), reason="Skip if no CUDA device")
+class CUDATester(Tester):
+    def setUp(self):
+        self.device = "cuda"
+
+
+if __name__ == '__main__':
```

Comment:
Not sure we need a separate test file for this once it's refactored to 1-2 unit-tests. You could add them to test_transforms_tensor.py.

Reply:
Should I create a function `test_assert_image_tensor` in test_transforms_tensor.py and add a few test cases there?

Reply:
That's a good idea. Also we just merged a PR that adds a few more operators. Would you mind making your changes to those as well?